packages/explorer-for-endevor/.gitignore

```
@@ -1,6 +1,2 @@
 dist
 .env
```

```suggestion
.env
README.md
CHANGELOG.md
LICENSE
NOTICE.md
```

I guess, it was mistakenly removed. We need to ignore autogenerated docs and other docs that are copied into E4E directory.